routes/multipart-form-data.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+const multer = require('koa-multer');
+const upload = multer({ dest: 'uploads/' });
+
+module.exports = ({ router }) => {
+  if (!fs.existsSync('uploads')) fs.mkdirSync('uploads');
```

Comment:
can this be more explicit? `path.resolve(__dirname, '..', 'uploads')`

right now it'll just make the directory at `.` wherever you are when you run the app.

Reply:
See 4ab21fe.